packages/core-entities/src/token-balance.ts

```diff
@@ -409,17 +409,38 @@ export class TokenBalance {
     decimalPlaces?: number,
     abbr = true,
     useThousandsAbbr = true,
-    locale = 'en-US'
+    locale = 'en-US',
+    hideSmallNegativeValues = false
   ) {
+    function containsNonZeroNumber(str: string) {
```

Comment:
Move this function declaration to the utils package instead. Inline function declarations incur a performance overhead.